agenda/presentation/src/main/java/com/gkp/agenda/presentation/compoments/EventPhotosSection.kt

```diff
@@ -0,0 +1,111 @@
+package com.gkp.agenda.presentation.compoments
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.horizontalScroll
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ExperimentalLayoutApi
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Add
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+import com.gkp.agenda.presentation.R
+import com.gkp.core.designsystem.theme.TaskyGray
+import com.gkp.core.designsystem.theme.TaskyLightGray
+
+@Composable
+fun EventPhotosSection(
+    modifier: Modifier = Modifier,
+    photosUris: List<String>,
+    onAddPhotoClick: () -> Unit,
+) {
```

Comment:
Minor - when creating composables, keep the following in mind:

1. `Modifier` should be the first _optional_ parameter in your parameter list. 
2. The last parameter should not be a non-Compose lambda.
3. If a Composable is only used in this file, mark it as `private`.

Not following **no 2**, you could write the following, which would be misleading as someone would expect this lambda block to be for more Compose code:

```
EventPhotosSection(
  modifier = Modifier.size(12.dp),
  // other parameters
) {
  // WRONG - handle onAddPhotoClick here
}
```